extensions/binding/org.eclipse.smarthome.binding.tradfri.test/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,25 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name:  Tradfri Binding Tests
+Bundle-SymbolicName: org.eclipse.smarthome.binding.tradfri.test;singleton:=true
+Bundle-Vendor: Eclipse.org/SmartHome
+Bundle-Version: 0.9.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Fragment-Host: org.eclipse.smarthome.binding.tradfri
+Import-Package: org.eclipse.smarthome.binding.tradfri,
+ org.eclipse.smarthome.binding.tradfri.handler,
+ org.eclipse.smarthome.core.common.registry,
+ org.eclipse.smarthome.core.events,
+ org.eclipse.smarthome.core.thing.util,
+ org.eclipse.smarthome.test.java,
+ org.eclipse.smarthome.test.storage,
+ org.hamcrest;core=split,
+ org.junit;version="4.0.0",
```

Comment:
On test fragments, we are usually not too strict regarding version constraints - they sometimes are needed. For junit, it makes sense to rule out that no JUnit 3 is used by mistake as this has a completely different API.